code/modules/shuttle/emergency.dm

```diff
@@ -262,7 +262,7 @@
 				if(shuttle_areas[get_area(player)])
 					has_people = TRUE
 					var/location = get_turf(player.mind.current)
-					if(!(player.mind.special_role == "traitor" || player.mind.special_role == "Syndicate") && !istype(location, /turf/open/floor/plasteel/shuttle/red) && !istype(location, /turf/open/floor/mineral/plastitanium/brig))
+					if(!(player.mind.special_role == "traitor" || player.mind.special_role == "Syndicate") && !istype(location, /turf/open/floor/metal/shuttle/red) && !istype(location, /turf/open/floor/mineral/plastitanium/brig))
```

Comment:
Just remove this. We don't use /shuttle paths anymore, just titanium and plastitanium for shuttle floors.